pom.xml

```diff
@@ -27,7 +27,7 @@
     <commons-collections4.version>4.4</commons-collections4.version>
     <folio-service-tools.version>3.1.0</folio-service-tools.version>
     <mapstruct.version>1.5.2.Final</mapstruct.version>
-    <lib-fqm-query-processor.version>1.0.0</lib-fqm-query-processor.version>
+    <lib-fqm-query-processor.version>modfqmmgr-87-SNAPSHOT</lib-fqm-query-processor.version>
```

Comment:
This should be `1.1.0-SNAPSHOT`, but I wanted to test against the other branch in CI.  Will update once the `lib-fqm-query-processor` PR is merged.